source/java/src/org/lucee/extension/cache/eh/EHCache.java

```diff
@@ -271,8 +271,9 @@ private static String[] createHash(Struct[] arguments) {
 		return hashes;
 	}
 
-	private static String createXML(String path, String cacheName,Struct arguments, String hash, RefBoolean isDistributed) {
-		//boolean isDistributed=false;
+	private static String createXML(String path, String cacheName,Struct arguments, String hash, RefBoolean isDistributed, Log LOG) {
+		LOG.debug("ehcache", "Building ehCache XML...");
```

Comment:
this should be lower case (LOG), upper case means it is a constant, what it is not.

Reply:
Fixed.